### ISSUE TITLE
bug: use stretch instead of fill-available

### ISSUE THREAD

**Issue by gabbloquet** [adeo/mozaic-design-system#1203]
# 🐞 Autoprefixer bug report

### Is this a regression?

Nope I don't think so.

### Description

By using [React plugin](https://github.com/adeo/mozaic-react), when I install it on a clean project like https://github.com/adeo/demo-dxp-react, I met an issue : 

`autoprefixer: Replace fill-available to stretch, because spec had been changed`

This come from [autoprefixer](https://www.npmjs.com/package/autoprefixer). 
In you opinion, using `stretch` instead of `fill-available` could be interesting ? 

I saw you are using autoprefixer so it could be interesting to see if them advise break the code. 

## 🔬 Steps to reproduce the issue

1. `npm install` on https://github.com/adeo/demo-dxp-react
2. `npm start`

## 📷 Screenshot (optional - it might be a gif)

![image](https://user-images.githubusercontent.com/25029077/189680583-7e6a403a-4e15-41e6-bb31-914be865cf41.png)

## 🌍 Your Environment

 - Ubuntu
 - node v16

**Anything else relevant?**

When your CSS input is :

```
.element {
    max-width: stretch;
}
```

**Autoprefixer will compile it to:**

```
.element {
    max-width: -webkit-fill-available;
    max-width: -moz-available;
    max-width: stretch;
}
```

**Comment by gabbloquet** [comment]
Oups maybe link to https://github.com/adeo/mozaic-design-system/pull/1194/files ?

**Comment by tiloyi** [comment]
Hi @gabbloquet ,
Thanks for your message.

That's right, this problem has already been fixed in PR #1194 and will be available in the next release.
